File: LeetCode/cpp/485.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int> &nums) {
        int m = 0 , maxM = 0;
        for (int i = 0; i < nums.size(); ++i) {
            if (nums[i] == 1) {
                ++m;
                maxM = max(maxM,m);
            }
            else
                m = 0;
        }
        return maxM;
    }
};

File: LeetCode/python3/485.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        consecutive = 0
        ret = 0
        for i in range(len(nums) + 1):
            if i == len(nums) or nums[i] == 0:
                ret = max(ret, consecutive)
                consecutive = 0
            else:
                consecutive += 1
        return ret
